pgpy/pgp.py

```diff
@@ -2343,24 +2349,35 @@ def verify(self, subject, signature=None):
             raise TypeError("Unexpected signature value: {:s}".format(str(type(signature))))
 
         def _filter_sigs(sigs):
-            _ids = {self.fingerprint.keyid} | set(self.subkeys)
-            return [ sig for sig in sigs if sig.signer in _ids ]
+            _ids = {self.fingerprint} | set(self.subkeys)
```

Comment:
this looks like an unrelated change -- why did change from `self.fingerprint.keyid` to `self.fingerprint` ?

Reply:
I don't remember now - more than half of year has passed. But I remember that I dislike the ways the keys are internally identified using short ids instead of full hashes and than I was trying to fix that. It may have been a part of another PR, but I have examined them and haven't found a suitable one.

Could you review and merge my other PRs too?